governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+- Reduce the WSTETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WSTETH-B Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the RETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WBTC-A Stability Fee (SF) by 0.11% from 5.80% to 5.69%
+- Reduce the WBTC-B Stability Fee (SF) by 0.11% from 6.30% to 6.19%
+- Reduce the WBTC-C Stability Fee (SF) by 0.11% from 5.55% to 5.44%
+
+#### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+As per [MIP104 ](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope) the debt ceiling (``line``) for ``CRVV1ETHSTETH-A`` will be set to ``0``. 
+
+This change will be happening alongside the other [Stability Scope Parameter changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6).
+
+
+### Deploy Multiswap Conduit for RWA015-A (Andromeda)
+
+Following the results of a recent [technical assessment from ChainSecurity](https://github.com/makerdao/rwa-toolkit/blob/8d30ed2cb657641253d45b57c894613e26b4ae1b/audits/ChainSecurity_MakerDAO_RWA_Toolkit_audit.pdf), the following changes will be made to [BlackTower Andromeda RWA015](https://github.com/makerdao/rwa-toolkit) if this proposal passes:
+
+- A new version of the output conduit will be added to the chainlog as `RWA015_A_OUTPUT_CONDUIT`. This will allow an authorized party to choose which PSM they want to interact with.
+- The operator on the old output conduit will be de-authorized.
+- `RWA015_A_OUTPUT_CONDUIT_LEGACY` will be removed from the chainlog.
+
+More context around [RWA015] Project Andromeda: Technical Assessment can be found [here](http://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974/11).
+
+### Deploy FlapperUniV2
+
+The Smart Burn Engine (Dss Flappers) allocates excess Dai from the Surplus Buffer and accumulates MKR tokens in the form of Univ2 LP tokens. Surplus Auctions (flaps) will be replaced by the [Dss Flappers](https://github.com/makerdao/dss-flappers/blob/univ2/README.md) contracts ([audit](https://chainsecurity.com/security-audit/maker-flapperuniv2/)).
+
+Given the successful launch [poll](https://vote.makerdao.com/polling/QmQmxEZp) the following actions will take place if this proposal passes: 
+
+* Deprecate MCD_FLAP and remove MCD_FLAP from chainlog
+* Deploy FlapperUniV2
+* Set ``hop`` to 1,577 seconds - hop defines the minimum interval between kicks, limiting the frequency of market actions.
+* Set ``want`` to 0.98 - A want of 0.98 means that the purchase price of MKR can be up to 2% worse than the oracle price.
+* Set ``receiver`` to [MCD_PAUSE_PROXY](https://etherscan.io/address/0xbe8e3e3618f7474f8cb1d074a26affef007e98fb) - destination address of the LP tokens purchased by the Smart Burn Engine.
+* Add FlapperJob and add to chainlog
+* Set ``pip`` to TBD
+* [Surplus Lot Size](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-lot-size): Reduce ``vow.bump`` by 25,000 DAI from 30,000 DAI to 5,000 DAI
```

Comment:
Generally I would recommend not using the smart-contract terms in the copy like this if an alternative exists. The Aliases are listed in the documents you are linking to, and as a general rule, it's better to phase out the contract names in favour of human-understandable names.

Reply:
I don't like the reduce by x from y to z structure for parameters like this. Patrick does. I think it just adds ways to confuse people in most cases. The exceptions are stability fees and DSR, liquidation ratio. Anything 'customer facing' should be structured like this. But for things like bump and hump that are primarily DAO facing, it's much less useful to include by 'how much' something is being reduced. I would go: 'reduce from x to y'